src/en/space-station-14/departments/security/proposals/reduced-metagaming.md

```diff
@@ -0,0 +1,234 @@
+ 
+The main point is the damage taken by the Implant Extractor. Performing an implant check without sufficient suspicion and being wrong becomes a very high price to pay, encouraging Security to be sure of their search. Even a single failed search essentially requires the user to be sent to the medbay, and two puts them out of commission for a long time.
+
+Note: The particulars of this suggestion may change as Newmed/Surgery gets implemented. The important part is that implant searching someone without being correct should punish Security by temporarily taking away a resource, in this case in the form of personnel. The exact nature of the damage and restriction on resources could change.
+
+## The PDA / Uplink Problem
+
+```admonish quote
+Confiscating PDAs of anyone you suspect of being a traitor without proof.
+```
+
+Uplinks are in a kind of weird situation where it's one of the most powerful tools for a Traitor and in-universe a very well-hidden feature, but at the same time it's also one of the easiest metaknowledge items in the game. As soon as a player has been found using a traitor-exclusive item Security players have to pretend that the PDA isn't a high target to confiscate. The reason for this is because you generally want antags to be able to use all their TC during a round to make it interesting. As such, the PDA may only be confiscated for two reasons:
+- Security finds an open uplink in the PDA.
+- Security has previously found an open uplink in another PDA, and can thus pre-emptively confiscate any PDA they have reasonable suspicion of having an uplink.
+
+Right now the rules are the main way of preventing overzealous Security confiscating PDAs. Traitors do have a gameplay option as well: The Uplink Implant. This implant does have the benefit of making PDA confiscating a non-issue, but the tradeoff is 2 less TC and an implanter that must be disposed of, and with the stringent rules on PDA confiscating it's unlikely to get any benefit from it. There's no reason to keep the uplink open after the implant is used, so you only get value if another uplink has been found. Additionally it suffers from the vulnerability of implant checks.
+
+### Proposed Solution
+
+The PDA is very easily the single point of failure with this problem, but it's also unique in its ringtone lock. Confiscating a PDA does come with some cost to Security in that they need to secure a replacement; worth it for a confirmed Syndie, but not viable for all arrests. If we were to remove the limitations on Security randomly confiscating PDAs there would need to be an equally effective method of eluding detection. It would also need to incur a cost for Security to confiscate. I propose the following:
+
+- Rule change: Security has knowledge PDAs can contain Uplinks, and may confiscate them if they have a reason to believe it could contain one.
+- Uplink Implanter cost is reduced to 1 TC, and changes proposed in "The Implant Problem" are also included.
```

Comment:
So, i think this completely breaks the original design intent behind things like the implanter, which was to be a way to evade losing your uplink *at cost of some of your TC.* Alongside that, I think the original design intent was made on sound grounds, with the idea being that the harder it is to fish out your uplink, the more TC you have to pay upfront to keep it.

Here's what I think might be a better solution:
- We introduce the ability to choose the type of uplink you want sometime around role application, with different uplink types having different TC costs.
- We keep PDAs as-is, but **allow you to pay a small amount of TC to transfer it to a new PDA** (likely by simply entering the dialtone into the new one, they're unique enough this should be fine.)
  - This makes keeping your original PDA highly desirable, as you would lose, say, 3-5TC, as a cost for setting up the new uplink, but *not* mandatory.
- We don't adjust uplink injector cost, it really needs to have a cost to it otherwise there's no reason to ever *not* use it, which makes the PDA uplink completely redundant.
- The uplink patch is a cool idea too, I don't think it should be discarded as a concept, but it also shouldn't be basically free (i.e. 1TC) to evade security that hard.
  - The uplink patch as described is decently powerful when used on stuff in your inventory, but where it's way more powerful is when used on random objects in maints or similar, allowing you to make it *impossible* to track down your uplink because it's actually a traffic cone in a locker somewhere. Essentially, it's way more powerful as a concept than this doc gives it credit for.

Reply:
Just wanna start by saying I appreciate the comprehensive feedback, really glad to see the topic get engaged!

I think that the main reason my suggestions regarding uplinks are so forgiving for antags is that I've leaned hard into the idea that confiscated TC is unfun. Sec should absolutely be rewarded if they find an uplink and similarly the Traitor should be punished for letting it be found, but if the uplink knowledge rules are relaxed there needs to be a safeguard against a Sec confiscating a PDA "because you had a syndie item and I know uplinks are in PDAs". 

Basically if you allow Security to know where locked uplinks may exist, you have two options. Either make it too difficult for Sec to just guess where the uplink is, or make it not punishing if Security takes your locked uplink. I really like your idea of being able to transfer the uplink to a new object, but I would really reduce the cost of it to 1 or 2 TC. That way you get the full punishment if you leave your PDA unlocked, but if Sec decides to be ornery and confiscate your locked PDA you aren't harshly punished for it.

You're right that the uplink patch does have more power than I gave it credit. However I would still keep it at a low TC. The reason is the same as what I give for the implant; it's highly situational, doesn't offer anything on its own and especially if you hide it in maints it's vulnerable to being grabbed by some random person. If Sec can know about how uplink works then Traitors must be given options for cheap to counteract it.

Reply:
I'm going to be very firm on the patch thing, it's *really not hard in my experience* to make it extremely unlikely someone will ever find an item even on an 80 player station, especially if secret rooms/etc are used.

This already technically makes hiding your uplink really easy if you're smart about it, and I've used tactics like that before to terrorize sec as a greytider because they can't find any of the junk I took from them. (Turning yourself in after hiding all your stuff in a room nobody ever visits is an extremely effective tactic.)

So apply that knowledge to a random, seemingly worthless item (say, a cig butt), and put it in a infrequently used room in maintenance somewhere. Nobody will ever find it, and you paid effectively nothing for that privilege.

Also, a thing I actually forgot to mention, most SS13 servers have *two* free uplinks you can choose from, the standard PDA uplink and the less common, but still cost free headset-based uplink. It'd likely be good to implement the headset-based uplink.

Reply:
I did consider a headset uplink and had it written out in an initial draft, but found that it bumped up against the issue of an overzealous Sec confiscating *both* the PDA and headset. I'm not opposed to it (I think it'd be a cool feature!) but because of that I rolled it into the patch suggestion. 

By the sound of things we're getting to the number tweaking part of implementation for the patch. Even if it ends up more expensive than what I consider optimal it still sounds like a good addition to the game.